source/documentation/deploying-an-app/use-circleci-to-upgrade-app.html.md.erb

```diff
@@ -23,14 +23,78 @@ By the end of the tutorial, you will have done the following:
 It is assumed you have the following:
 
  - You have [created an environment for your application][env-create]
- - You have [deployed an application][deploy-helm] to the 'cloud-platform-live-1' cluster using Helm.
+ - You have deployed the multi-container rails app to the 'cloud-platform-live-1' cluster.
  - You have created an [ECR repository][ecr-setup]
 
 ### Creating a Service Account for CircleCI
 As part of the CircleCI deployment pipeline, CircleCI will need to authenticate with the Kubernetes cluster. In order to do so, Kubernetes uses [Service Accounts](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/). Service Accounts provide an identity for processes that run in a cluster allowing the process to access the API server.
 
 A Service Account is created in the [namespace creation github repository](https://github.com/ministryofjustice/cloud-platform-environments/tree/master/namespaces).
 
+If you have not created a Circle CI service account along with the role / rolebinding then you may
+use the below as an example:
+
+```bash
```

Comment:
using `'''yaml` instead of `'''bash` results in more consistent and useful syntax highlighting for yaml examples

Reply:
done.